docs/src/modules/ROOT/pages/using-timefold-solver/modeling-planning-problems.adoc

```diff
@@ -545,9 +545,6 @@ One way to deal with this is to filter the entity selector of the placer in the
 A planning variable is considered initialized if its value is not `null` or if the variable is ``allowsUnassigned``.
 So a variable which allows unassigned values is always considered initialized.
 
-A planning entity is initialized if all of its planning variables are initialized.
```

Comment:
Does it no longer make sense?

Reply:
For variables, not really. I moved the text to a section about entities, and slightly improved it.